featuretools/io/sql.py

```diff
@@ -0,0 +1,79 @@
+import pandas as pd
+import sqlalchemy as sa
+
+import featuretools as ft
+
+
+def from_sql(id, connection):
+    loader = EntitySetLoaderFromSQL(connection=connection, id=id)
+    loader.load_all_tables()
+    loader.load_relationships()
+    return loader.es
+
+
+def _find_pk_name(table):
+    column_names = table.columns.keys()
+    pks = [name for name in column_names if table.columns[name].primary_key]
+    if len(pks) > 1:
+        raise RuntimeError('Composite primary key detected')
+    elif len(pks) == 1:
+        return pks[0]
+    else:
+        return None
+
+
+class EntitySetLoaderFromSQL:
+    def __init__(self, id, connection):
+        self.connection = connection
+        self.metadata = sa.MetaData(connection)
+        self.metadata.reflect(bind=self.connection.engine)
+        self.table_names = self._get_table_names()
+        self.es = ft.EntitySet(id=id)
+
+    def _get_table_names(self):
+        return list(self.metadata.tables.keys())
+
+    def _load_table(self, table_name):
+        table = sa.Table(table_name, self.metadata, autoload=True)
+        index_name = _find_pk_name(table)
+        make_index = index_name is None
+        if make_index:
+            index_name = 'featuretools_sql_import_id'
```

Comment:
I'm supposing it is best to generate an index in the case where we don't detect a pk - thoughts about this or the naming convention?